src/app/components/NearBy/NearBy.tsx

```diff
@@ -38,46 +38,29 @@ export default function NearSection() {
           </h2>
         </div>
         <div className={styles.flexWrap}>
-          <div className={`${styles.leftSection} ${styles.leftSectionLg}`}>
+          <div className={styles.leftSection}>
             {nearBy.map(([time, location], index) => (
               <div key={index} className={styles.nearByItem}>
                 <div className={styles.itemWrapper}>
-                  <div
-                    className={`${styles.timeBox} ${styles.timeBoxLg} ${styles.timeBoxMd}`}
-                  >
-                    <p
-                      className={`${styles.timeText} ${styles.timeTextMd} ${styles.timeTextSm}`}
-                    >
-                      {time}
-                    </p>
-                    <span
-                      className={`${styles.minutesText} ${styles.minutesTextMd}`}
-                    >
-                      minutes
-                    </span>
+                  <div className={styles.timeBox}>
+                    <p className={styles.timeText}>{time}</p>
+                    <span className={styles.minutesText}>minutes</span>
                   </div>
 
-                  <div
-                    className={`${styles.locationBox} ${styles.locationBoxMd}`}
-                  >
-                    <p
-                      className={`${styles.locationText} ${styles.locationTextMd}`}
-                    >
-                      {location}
-                    </p>
-                  </div>
+                  <p className={styles.locationBox}>{location}</p>
                 </div>
               </div>
             ))}
           </div>
 
-          <div className={`${styles.rightSection} ${styles.rightSectionLg}`}>
-            <div className={`${styles.mapWrapper} ${styles.mapWrapperLg}`}>
+          <div className={styles.rightSection}>
+            <div>
               <iframe
                 src={locationUrl}
```

Comment:
@Dheeru12503 
check console this is generating error.
just fix this and all good

Reply:
this two line is not showing in my code.

Reply:
issues is coming in 
 <iframe
                src={locationUrl || "about:blank"}  // it error problem 
                width="100%"
                allowFullScreen
                className={styles.mapWrapper}
                loading="lazy"
              ></iframe>
              
              but is fix by adding this "about:blank"